student/views.py

```diff
@@ -79,14 +79,9 @@ def update_student(request, pk):
     elif request.method == 'GET':
         form = StudentCreateForm(instance=student)
 
-    form_html = f"""
-    <form method="POST">
-      {form.as_p()}
-      <input type="submit" value="Save">
-    </form>
-    """
-
-    return HttpResponse(form_html)
+    return render(
```

Comment:
Better to have a separate template from update funtionality.

Reply:
Button should be "Update"
![image](https://user-images.githubusercontent.com/17034253/138166259-4857b4b7-6624-441a-ae01-a64afd5f41a7.png)